awstest/modules/userpool/module.ftl

```diff
@@ -0,0 +1,92 @@
+[#ftl]
+
+[@addModule
+    name="userpool"
+    description="Testing module for the aws userpool component"
+    provider=AWSTEST_PROVIDER
+    properties=[]
+/]
+
+
+[#macro awstest_module_userpool ]
+
+    [#-- Base setup --]
+    [@loadModule
+        settingSets=[
+            {
+                "Type" : "Settings",
+                "Scope" : "Accounts",
+                "Namespace" : "mockacct-shared",
+                "Settings" : {
+                    "Registries": {
+                        "docker": {
+                            "EndPoint": "123456789.ecr.awsamazon.com"
+                        }
+                    }
+                }
+            },
+            {
+                "Type" : "Builds",
+                "Scope" : "Products",
+                "Namespace" : "mockedup-integration-aws-userpool-base",
+                "Settings" : {
+                    "COMMIT" : "123456789#MockCommit#",
+                    "FORMATS" : ["docker"]
+                }
+            }
+        ]
```

Comment:
Shouldn't need the build references for the userpool

Reply:
Sorry, copy-paste left-overs.